bin/setup.dart

```diff
@@ -28,13 +29,40 @@ void extract(String fileName, [String dir = '']) {
   }
 }
 
+/// Places downloaded mobile binaries to proper location.
+void placeMobileBinaries(String platform, String fileName) {
+  final android =
+      '${findPackageRoot().toFilePath(windows: Platform.isWindows)}/android';
+  Directory('$android/libs').createSync();
+  Directory('$binaryStorageDir/$platform/libs').listSync().forEach((jar) {
+    if (jar is File) {
+      jar.renameSync('$android/libs/${basename(jar.path)}');
+    }
+  });
+  Directory('$android/src/main/jniLibs').createSync();
```

Comment:
You should ideally pass `recursive: true` to `createSync` since this code will break incase the user doesn't have a directory already present (Although that might be unlikely).

Reply:
I think, in that case, shouldn't it just fail? As this folder structure is provided by Flutter as a boilerplate and a must have to get android support. :)